nats/aio/client.py

```diff
@@ -1023,23 +1023,21 @@ async def request(
             msg = await self._request_new_style(
                 subject, payload, timeout=timeout, headers=headers
             )
-        if msg.headers and msg.headers.get(nats.js.api.Header.STATUS
-                                           ) == NO_RESPONDERS_STATUS:
-            raise errors.NoRespondersError
```

Comment:
removing this here would make `await nc.request` not handle no responders correctly no?